Code/Framework/AzToolsFramework/Tests/TransformComponent.cpp

```diff
@@ -1063,4 +1066,61 @@ R"DELIMITER(<ObjectStream version="1">
             }
         }
     }
+
+    // Fixture provides a root prefab with Transform component and listens for TransformNotificationBus.
+    class TransformComponentActivationTest
+        : public PrefabTestFixture
+        , public TransformNotificationBus::Handler
+    {
+    protected:
+        void SetUpEditorFixtureImpl() override
+        {
+            PrefabTestFixture::SetUpEditorFixtureImpl();
+
+            CreateRootPrefab();
+        }
+
+        void TearDownEditorFixtureImpl() override
+        {
+            BusDisconnect();
+            
+            PrefabTestFixture::TearDownEditorFixtureImpl();
+        }
+        
+        void OnTransformChanged(const Transform& /*local*/, const Transform& /*world*/) override
+        {
+            m_transformUpdated = true;
+        }
+
+        void MoveEntity(AZ::EntityId entityId)
+        {
+            AzToolsFramework::ScopedUndoBatch undoBatch("Move Entity");
+            TransformBus::Event(entityId, &TransformInterface::SetWorldTranslation, Vector3(1.f, 0.f, 0.f));
+        }
+        
+        bool m_transformUpdated = false;
+    };
+    
+    TEST_F(TransformComponentActivationTest, TransformChanged_OnUndoRedo)
```

Comment:
nit: Test name as mentioned in other PR - https://github.com/o3de/o3de/pull/5674

Reply:
fixed